### ISSUE TITLE
Changing the Resolution for Fold 5?

### ISSUE THREAD

**Issue by codearm2** [Keriew/augustus#993]
Great mod!
However I need to ask something , I ran Augustus on my Fold 5 which use 1812 x 2176 px resolution and things get a little bit blurry

Is there anyway to maximize the pixel setting?

**Comment by crudelios** [comment]
Hi,

For readability reasons, android phones auto apply a reasonable zoom.

That zoom can only be changed if the screen is large enough to allow a different scale without issues.

Try unfolding the phone, playing on landscape and going to the user interface options menu. A "display scale" setting should appear, which if set to 100% or a multiple of 100 won't be blurry.

**Comment by codearm2** [comment]
I cant find the option, even when unfolded and landscape mode

**Comment by crudelios** [comment]
Yeah it should work? I'm not sure as I wasn't the one to implement scaling on Android.

Can you check if Julius works, please? The original code change was there, I may have messed something up when I ported the changes.
